tests/testsuite/inheritable_workspace_fields.rs

```diff
@@ -1589,7 +1591,9 @@ fn cannot_inherit_in_patch() {
         .with_stderr(
             "\
 [WARNING] [CWD]/Cargo.toml: unused manifest key: patch.crates-io.bar.workspace
-[WARNING] [CWD]/Cargo.toml: dependency (bar) specified without providing a local path, Git repository, or version to use. This will be considered an error in future versions
+[WARNING] [CWD]/Cargo.toml: dependency (bar) specified without providing a local path, Git repository, version, \
```

Comment:
This warning message is quite confusing. 
It is not an empty dep:
```toml
[workspace.dependencies]
bar = { path = "bar" }
```

But the warning said it is.

Reply:
Is there any existing issue, or could you submit one?